tests/Jobs/RockTheVote/ImportRockTheVoteRecordTest.php

```diff
@@ -12,6 +12,14 @@
 
 class ImportRockTheVoteRecordTest extends TestCase
 {
+    /**
+     * Set or unset the Update User SMS Subscription feature flag.
+     */
+    private function enableUpdateUserSmsFeature(bool $shouldUpdate)
+    {
+        \Config::set('import.rock_the_vote.update_user_sms_enabled', $shouldUpdate ? 'true' : 'false');
```

Comment:
What does the `$shouldUpdate` variable do? Does it store the boolean value?

Reply:
We're passing the boolean `$shouldUpdate` argument to this function to determine whether our environment variable should be set to 'true'` and `'false'`, because we can't set environment variables as boolean. This could have been written to expect the string variables :
```
  private function enableUpdateUserSmsFeature(string $envValue)
    {
        \Config::set('import.rock_the_vote.update_user_sms_enabled', $envValue);
    }
```
But it seemed easier to call the function by passing a boolean vs sending `'true'` and `'false'` strings. Hope that makes sense?

Reply:
ah! ok makes sense, thanks @aaronschachter